app/views/teams/TeamInfoPage.scala.html

```diff
@@ -196,13 +196,15 @@ <h3 class="board__heading">Team Members</h3>
 @showRepositories(repos: Seq[GitRepository], headerName: String, typeName: String) = {
   <div class="col-md-3">
       <div id="@headerName" class="board">
-          <h3 class="board__heading">@headerName</h3>
+          <abbr style="text-decoration: none; border-bottom: none;" title="Shared repos (owned by 8+ teams), are at the bottom of the list">
```

Comment:
if we add `(shared)` or something after the repo name, does it avoid the need for this help?